bin/setup

```diff
@@ -12,18 +12,10 @@ Dir.chdir APP_ROOT do
   system "gem install bundler --conservative"
   system "bundle check || bundle install"
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   system "cp config/database.yml.sample config/database.yml"
-  # end
-
   puts "\n== Preparing database =="
-  system "bin/rake db:setup"
+  system "bin/rake db:create db:setup db:structure:load"
```

Comment:
`rake db:setup` should be all you need - it calls `db:create db:structure:load db:seed`

Reply:
@pixeltrix Good point. Done!